tutorials/python/python-video-generation/tutorial.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Dynamic Video Generation with Python
+published: 2019-04-01
+lts: 3.6.5
+tags: python, video, multimedia, dynamic media
+libraries: moviepy, gizeh
+language: Python
+author-name: Sebastian Arias
+twitter-profile: sebasarias95
+github-profile: larox
+description: Python is a language that can be use for everything, generating dynamic content like a video is a good and fun way to get started in Python.
+---
+
+
+# Video Generation with Python
+___
+
+
+When you hear about python, you might have in mind the powerful interpreted programming language used for data science, artificial intelligence and for web apps with flask or Django. But Python is not only capable of achieving all of this, here in SB we’ve found that is possible to create and render a fully customized video with animations, text and draw vectors using python’s libraries moviepy and gizeh.
```

Comment:
I have a suggestion regarding the wording:
`we've found that` sounds kind of weird to me. 

> here in SB we’ve found that is possible to create and render a fully customized video with animations
here in SB, we’ve also used it to create and render a fully customized video with animations

In any case, it's probably worth to check it with Andrea

Reply:
I double checked with Andrea and your suggestion was right! Thanks @juanpaucar !